tree/dataframe/inc/ROOT/RDFHelpers.hxx

```diff
@@ -92,6 +101,39 @@ auto PassAsVec(F &&f) -> RDFInternal::PassAsVecHelper<std::make_index_sequence<N
 {
     return RDFInternal::PassAsVecHelper<std::make_index_sequence<N>, T, F>(std::forward<F>(f));
 }
+template <typename Proxied, typename DataSource>
+class RInterface;
+
+// clang-format off
+/// Creates a file with the dot representation of the graph.
+/// \param[in] node any node of the graph. If the node is a LoopManager, it prints the entire graph. Otherwise, only the branch the node belongs to.
+/// \param[in] filePath where to save the representation.
+// clang-format on
+template <typename NodeType>
+void Show(NodeType& node, std::string filePath){
```

Comment:
coding conventions

Reply:
string_view instead of string (by copy!)

Reply:
filePath --> dotFilePath

Reply:
and then dotFilePath="" without the overload.

Reply:
Would the name rather be `CreateGraphDescription`?

Reply:
or `RepresentGraph`?

Reply:
ostream does not accept a string_view, it has to be converted to string anyway. I should just pass it by const ref